lib/portfinder.js

```diff
@@ -157,10 +168,15 @@ exports.getPort = function (options, callback) {
   }
 
   var candidatePorts = []
-  for (i = options.port; i < options.stopPort; i++) {
+  for (i = options.port; i <= options.stopPort; i++) {
     candidatePorts.push(i)
   }
-  randomShuffle(candidatePorts)
+
+  if (options.useRandom) {
+    randomShuffle(candidatePorts)
+  } else {
+    candidatePorts.reverse()
```

Comment:
Why is the array reversed? Are they not tested in order afterwards? Were they not tested in order before?